src/test/resources/log4j.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  Copyright (C) 2020 Expedia, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!DOCTYPE log4j:configuration SYSTEM "log4j.dtd">
+<log4j:configuration xmlns:log4j='http://jakarta.apache.org/log4j/'>
+
+  <appender name="CONSOLE" class="org.apache.log4j.ConsoleAppender">
+    <layout class="org.apache.log4j.PatternLayout">
+      <param name="ConversionPattern" value="%-4r [%t] %-5p %c %x - %m%n" />
+    </layout>
+  </appender>
+
+  <root>
+    <level value="DEBUG" />
+    <appender-ref ref="CONSOLE" />
+  </root>
+
+  <logger name="org.apache.hadoop.hive.metastore.HiveMetaStore.audit">
+    <level value="ERROR" />
+  </logger>
+
+  <logger name="org.apache.hadoop.hive.metastore.MetaStoreDirectSql">
+    <level value="ERROR" />
+  </logger>
+
+  <logger name="org.apache.hadoop.hive.metastore.ObjectStore">
+    <level value="ERROR" />
+  </logger>
+
+  <logger name="org.apache.hadoop.util.NativeCodeLoader">
+    <level value="ERROR" />
+  </logger>
+  
+  <logger name="org.apache.hadoop.hive.ql.log.PerfLogger">
+    <level value="ERROR"/>
+  </logger>
+
+  <logger name="org.apache.hadoop.security.UserGroupInformation">
+    <level value="ERROR"/>
+  </logger>
+
+  <logger name="org.apache.hadoop.conf.Configuration">
+    <level value="ERROR"/>
+  </logger>
+
+  <logger name="DataNucleus">
+    <level value="ERROR" />
+  </logger>
+
+</log4j:configuration>
```

Comment:
Add a [closing line break](https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file) to EOF please.

Reply:
Putting this back to the original one because it now seems to work in IntelliJ...